docs/usage/shoot_workerless.md

```diff
@@ -0,0 +1,29 @@
+# Workerless Shoot
+
+Starting from Gardener version v1.71, users can create a shoot without any workers, known as a "workerless shoot". Previously, users were unable to create shoots without workers, and had to include worker nodes even if they only needed control-plane services. With workerless shoots, Gardener will not create a worker plane.
+
+Here's an example manifest for a workerless shoot:
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+metadata:
+  name: local
+  namespace: garden-local
+spec:
+  seedName: local
+  cloudProfileName: local
+  region: local
+  provider:
+    type: local
+  kubernetes:
+    version: 1.26.0
+```
```

Comment:
Probably you should mention that this is very specific to the local scenario. Also, drop the `.spec.seedName` (this should be assigned by scheduler).